### PR TITLE
[junio]: Set default values for docker-compose and enhance .env.example

### PR DESCRIPTION
**Pull request by junio256**
Fixed the issue where Docker Compose wasn't loading default values from `.env.example` as expected.
Now added default values to cover cases with no loaded environment variables. Also, cleaned up strange `db.` prefixes in `.env.example.`

### CODE REVIEW

**Comment by MiguelMachado-dev** [comment]
It was not supposed to load from .env.example
.env.example as the name mention, it's an example.
You should create a .env file based on .env.example.

On README it is mentioning this as well. It's a common practice while using dotEnv

**Comment by MiguelMachado-dev** [comment]
Also, the values of .env.example is just a placeholder, you should change/replace according to what you like.

it's not a pattern to have `-db` for example. For instance, you changed docker-compose to load the variable and added `-db` to the end of the string. This is not the idea. The idea is to load the data from the variable and that's it!

TL;DR
`.env.example` is a common practice to push to repository so everyone know which variables are needed and .env by default load those values from `.env` file, or for example, on NextJS it loads from `.env.local`. That's why it was not loading from `.env.example` and that was expected.

Edit:
The `-db` you added was for adding if variable is nil, right? So disconsider the first part

**Comment by junio256** [comment]
> It was not supposed to load from .env.example

I believe that by having a default file simplifies the setup process for those who are only interested in testing the functionality but understandable
And with docker-compose require some values to be filled otherwise Postgres crashes

**Comment by MiguelMachado-dev** [comment]
> I believe that by having a default file simplifies the setup process for those who are only interested in testing the functionality but understandable

Not a big of a deal doing  `cp .env.example .env` and as I mentioned, is a common practice, you are going to face this a lot out there, so I think we can follow it. It's really pretty common.
It crashes because it needs the value to run, so when you have the correct file it will work just fine
